addons/web_editor/views/snippets.xml

```diff
@@ -243,11 +243,23 @@
                     <we-button data-shape="web_editor/Origins/14" data-select-label="Origins 14"/>
                     <we-button data-shape="web_editor/Origins/15" data-select-label="Origins 15"/>
                 </we-select-page>
-                <we-select-page string="Blocks">
+                <we-select-page string="Blocks &amp; Blobs">
```

Comment:
The two pages which had to be mixed are Blocks and Rainy, not Blocks and Blobs :wink: Correct me if I am wrong @JKE-be @sbuhl I tested and I am actually not convinced (makes more sense than Blobs but it's not that obvious with Rainy either)

Reply:
@qsm-odoo On the pad we wrote `Merge 2/4`, so Blocks and Blobs :-) 
And move 'Bold' after 'Airy'. 
I'm not sure about the merge though... I've asked a key user a confirmation.

Reply:
I know this is what is noted, but this is not what was decided obviously ;) It is an error in the pad

Reply:
@qsm-odoo @bso-odoo 
About the merge, I got the answer from the key user, we can merge Zigs et Blocks (2 and 6).
Thank you.

Reply:
Je confirme on a dit: le 2 (BLOCKS) et lé 5 (RAINY) !
Et on en rajoute dans le Zigs plus tard !